infra/README.md

```diff
@@ -1,8 +1,25 @@
-## Fetching Debian Bullseye Image and Verifying It
 
-- `./base-image/fetch-debian-image.sh`
+## Environment choice
+This section covers how to set a production-like environment. 
+For most developments, it is better to use the containers instead (see the dev-env repository). The infrastructure environment provides additional services like reverse-proxy (nginx) and monitoring, but is also much less practical.
```

Comment:
"less practical", it depends for what. It's meant to test infrastructure modifications and software modification in a prod-like environment. It's less practical while developing because you need to redeploy instead of having hot reload.